model/application_authentication_http.go

```diff
@@ -1,13 +1,9 @@
 package model
 
-import (
-	"time"
-)
-
 type ApplicationAuthenticationResponse struct {
-	ID        string    `json:"id"`
-	CreatedAt time.Time `json:"created_at"`
-	UpdatedAt time.Time `json:"updated_at"`
+	ID        string  `json:"id"`
+	CreatedAt *string `json:"created_at"`
+	UpdatedAt *string `json:"updated_at"`
```

Comment:
Like mikel said i think created/updated at are always going to be there due to gorm so they don't have to be pointers.

Reply:
done